R/coveralls.R

```diff
@@ -57,9 +71,45 @@ to_coveralls <- function(x, service_job_id = Sys.getenv("TRAVIS_JOB_ID"),
       "source_files" = res)
   }
 
+  if (!is.null(git_info)) payload <- c(payload, git = list(git_info))
```

Comment:
this can just be

```r
payload$git <- git_info
```

if git_info is `NULL` then it won't be added to the list

Also what happens when repo token is not null but git_info is?  Payload won't be declared on line 74